lib/libc/stdlib/lib_abs.c

```diff
@@ -63,8 +63,9 @@
 
 int abs(int j)
 {
-	if (j < 0) {
-		j = -j;
-	}
-	return j;
+	int mask;
+
+	mask = (j >> ((sizeof(j)*8) - 1));
```

Comment:
please add space between operator and operand

Reply:
thanks for the review!. I guess, I will wait for #978 to merged before amending this with your comments.